### ISSUE TITLE
Extraction of confidence intervals in STRUCTURE

### ISSUE THREAD

**Issue by PhilGossieaux** [EricArcher/strataG#10]
Hello, thanks for the very nice package. 
Could it be possible to extract the confidence intervals when running STRUCTURE ? It corresponds to the "credible regions" in the parameters. 
Thanks a lot.

**Comment by EricArcher** [comment]
Yes, it is totally possible and a good idea. I'll add this to the list of enhancements for the next round of updates. Thanks for the suggestion!

**Comment by PhilGossieaux** [comment]
Thanks a lot, I also wondered if there is a possibility to use these intervals in CLUMPP. Could it be possible to calculate the mean of the intervals just like we can calculate the mean of the q-values? 
Thanks!

**Comment by EricArcher** [comment]
It would definitely be possible to calculate the mean of the LCI and UCI over multiple runs. Depending on the shape of the distributions though, that might not be an appropriate summary of the intervals. I'll definitely take a look at it though when I sit down to do this.
